.orchestra/support/revng-distributable/test.sh

```diff
@@ -22,6 +67,10 @@ if test -e "$TARGET"; then
   cd "$TARGET"
 fi
 
+if [[ -n "$USE_XVFB" ]]; then
```

Comment:
This assumes `USE_XVFB=0` enables xvfb.

```suggestion
if test "$USE_XVFB" -eq 1; then
```

Reply:
Also good point